staging/src/kubevirt.io/api/go.mod

```diff
@@ -1,16 +1,14 @@
 module kubevirt.io/api
 
-go 1.22.0
-
-toolchain go1.22.2
+go 1.23.0
```

Comment:
I guess you have run the `go work sync` can you just mention it in the commit?

Reply:
Added a comment to the commit's description.